### PR TITLE
🌐 Add Spanish traslation for Body - Fields

### PR DESCRIPTION
**Pull request by anthonycepeda**
Hi Guys :)

According to Spanish Traslation Feature: #1218 this PR includes the spanish traslation for `tutorial/body-fields.md`

Thanks

### CODE REVIEW

File: docs/es/docs/tutorial/body-fields.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Los Campos del Cuerpo (Body - Fields)
+
+De la misma forma que puedes declarar parámetros con `Query`, `Path` y `Body` para realizar validaciones adicionales y de metadatos en la ruta de operacion (_path operation_) de una función, es posible realizar dichas acciones desde los modelos de Pydantic usando `Field`.
```
Comment (bezaca) [inline docs/es/docs/tutorial/body-fields.md:3]:
```suggestion
De la misma forma que puedes declarar parámetros con `Query`, `Path` y `Body` para realizar validaciones adicionales y de metadatos en la <abbr title="path operation">ruta de operacion</abbr> de una función, es posible realizar dichas acciones desde los modelos de Pydantic usando `Field`.
```

Comment (bezaca) [reply docs/es/docs/tutorial/body-fields.md:3]: Hi there, I suggested to use the abbr tag for the translations

File: docs/es/docs/tutorial/body-fields.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+`Query`, `Path` y otros objetos que verás mas adelante, son en realidad subclases de una clase común llamada `Param`, que a su vez es una subclase de la clase `FieldInfo` de Pydantic.
+
+    De la misma forma, el objeto `Field` devolverá una instancia de `FieldInfo`.
+
+    Igual que otros que verás mas adelante, `Body` también devuelve objetos que son subclase de `FieldInfo`.
+
+    Recuerda que cuando importas `Query`, `Path`, y otros objetos desde `fastapi`, éstos son en realidad funciones que devuelven clases especiales.
+
+!!! tip
+Observa cómo el atributo de cada modelo con un tipo, valor por defecto y `Field`, tiene la misma estructura que el parámetro de una función de operación de ruta (_path operation_), con `Field` en lugar de `Path`, `Query` y `Body`.
```
Comment (bezaca) [inline docs/es/docs/tutorial/body-fields.md:36]:
```suggestion
Observa cómo el atributo de cada modelo con un tipo, valor por defecto y `Field`, tiene la misma estructura que el parámetro de una función de <abbr title="path operation">operación de ruta</abbr>, con `Field` en lugar de `Path`, `Query` y `Body`.
```

File: docs/es/docs/tutorial/body-fields.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+Puedes declarar información adicional usando `Field`,` Query`, `Body`, etc. Y ésta será incluida en el esquema JSON generado.
+
+Aprendrás mas sobre cómo añadir información extra más adelante, cuando veas cómo declarar ejemplos.
+
+## Resumen
+
+Puedes usar `Field` de Pydantic para declarar validaciones y metadatos adicionales para los atributos del modelo.
+
+También puedes usar los argumentos de palabras clave (_keyword arguments_) para pasar metadatos de esquema JSON adicionales.
```
Comment (bezaca) [inline docs/es/docs/tutorial/body-fields.md:48]:
```suggestion
También puedes usar los argumentos de <abbr title="keyword arguments">palabras clave</abbr> para pasar metadatos de esquema JSON adicionales.
```

**Review by bezaca — APPROVED** [review]

File: docs/es/docs/tutorial/body-fields.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+`Field` funciona de la misma forma que `Query`, `Path` y `Body`, y además cuenta con los mismos parámetros.
+
+!!! note "Detalles Técnicos"
+`Query`, `Path` y otros objetos que verás mas adelante, son en realidad subclases de una clase común llamada `Param`, que a su vez es una subclase de la clase `FieldInfo` de Pydantic.
+
+    De la misma forma, el objeto `Field` devolverá una instancia de `FieldInfo`.
+
+    Igual que otros que verás mas adelante, `Body` también devuelve objetos que son subclase de `FieldInfo`.
```
Comment (AdrianDeAnda) [inline docs/es/docs/tutorial/body-fields.md:31]:
```suggestion
    `Body` también devuelve objetos que son sublcase de `FieldInfo` directamente. Y hay otros que se verán más adelante que son subclases de la clase `Body`. 
```

File: docs/es/docs/tutorial/body-fields.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Los Campos del Cuerpo (Body - Fields)
+
+De la misma forma que puedes declarar parámetros con `Query`, `Path` y `Body` para realizar validaciones adicionales y de metadatos en la <abbr title="path operation">ruta de operacion</abbr> de una función, es posible realizar dichas acciones desde los modelos de Pydantic usando `Field`.
+
+## Importa `Field`
+
+Primero, importa Field:
```
[the remark this answers is not in the document]
Comment (anthonycepeda) [reply docs/es/docs/tutorial/body-fields.md]: Thanks @AdrianDeAnda for the suggestions

**Comment by tiangolo** [comment]
This was covered in another PR, so I'll close this one now, thanks for the effort! :cake: :coffee:

**Pull request closed without merging**